paint/PAINT_ButtonFunctions.h

```diff
@@ -7,14 +7,18 @@ namespace win {
 }
 
 namespace paint {
-
 	void toggleDraw(win::UIelement* control);
+	void toggleDrawRectangle(win::UIelement* control);
+	void toggleDrawEllipse(win::UIelement* control);
+	void toggleDrawTriangle(win::UIelement* control);
 	static void setBrushThickness(win::UIelement* control, int thick);
 	void setBrushThickness0(win::UIelement* control);
 	void setBrushThickness1(win::UIelement* control);
 	void setBrushThickness2(win::UIelement* control);
 	void swapColours(win::UIelement* control);
 	void clearScreen(win::UIelement* control);
+
+	static win::ToggleButton* CastToggleButton(win::UIelement* control);
```

Comment:
As above, but static functions shouldn't be mentioned in header files. However, static methods of classes must be, so be careful not to confuse the two!

Move function prototype into cpp file, drop the namespace.
**SHOULD**

Reply:
done

Reply:
Could you please explain the difference between static functions and static methods of classes?

Reply:
done